source/install/compose_guide.rst

```diff
@@ -115,3 +115,10 @@ is, the machine where docker-compose was run). The script is located at the root
 ::
 
   ./create.user.sh
```

Comment:
I'd remove this section.

Reply:
I've other change to do it, but I did not push it yet because it makes tag 0.1.0 inconsistent.

Reply:
As we have no tagged documentation for 0.1.0 version, you're right.